content/actions/migrating-to-github-actions/manual-migrations/migrating-from-travis-ci-to-github-actions.md

```diff
@@ -318,7 +318,7 @@ This section compares how {% data variables.product.prodname_actions %} and Trav
 
 You can create custom environment variables in a {% data variables.product.prodname_actions %} job.
 
-#### Travis CI syntax for an environment variable
+### Travis CI syntax for an environment variable
```

Comment:
I actually think h4 is correct for this one, as it's a subsection of "Configuring environment variables". Our mistake!

```suggestion
#### Travis CI syntax for an environment variable
```